fighters/dedede/src/status.rs

```diff
@@ -3,39 +3,26 @@ use globals::*;
 
 // Prevent going into air jet hammer when Special is released during Jumpsquat
 
-unsafe extern "C" fn special_lw_jump_squat_exec(fighter: &mut L2CFighterCommon) -> L2CValue{
-    if ControlModule::check_button_off(fighter.module_accessor, *CONTROL_PAD_BUTTON_SPECIAL){
-        StatusModule::change_status_force(fighter.module_accessor, *FIGHTER_DEDEDE_STATUS_KIND_SPECIAL_LW_ATTACK, true);
+unsafe extern "C" fn special_lw_jump_squat_exec(agent: &mut L2CFighterCommon) -> L2CValue{
```

Comment:
Statuses go in their own individual folders (status/special_lw_jump_squat.rs for example)